elements/lisk-transactions/src/utils/validation/validation.ts

```diff
@@ -35,6 +36,15 @@ export const validatePublicKey = (publicKey: string) => {
 	return true;
 };
 
+export const validateNetworkIdentifier = (networkIdentifier: string) => {
+	const networkIdentifierBuffer = cryptography.hexToBuffer(networkIdentifier);
+	if (networkIdentifierBuffer.length !== MAX_NETWORK_IDENTIFIER_LENGTH) {
```

Comment:
Why we are using the prefix `MAX` in the constant name if it checks the exact length?